<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Peter Zhou and Ian Monroe</author>
 <class>ScriptsConfig</class>
 <widget class="QWidget" name="ScriptsConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>115</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ScriptSelector" name="scriptSelector" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="uninstallButton">
       <property name="text">
        <string>Uninstall Script</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="installButton">
       <property name="text">
        <string>Install Local Script</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="m_scriptConsoleGroup">
     <property name="title">
      <string>Script Console</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="kcfg_EnableScriptConsole">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="statusTip">
         <string>Enable the script console.</string>
        </property>
        <property name="whatsThis">
         <string>Enable the script console.</string>
        </property>
        <property name="text">
         <string>Enable Script Console</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="0"/>
 <customwidgets>
  <customwidget>
   <class>QPushButton</class>
   <extends>QPushButton</extends>
   <header>qpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>ScriptSelector</class>
   <extends>QWidget</extends>
   <header>configdialog/dialogs/ScriptSelector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
